web/components/lib/templates/priorityclass.yaml

```
@@ -0,0 +1,5 @@
+metadata:
+  name: default-priority-class
+value: 1000
+globalDefault: true
+description: "Default priority class for all pods"
```

nits: If my English is strange, please change it to a proper English sentence. (I'm not a native speaker.)

```suggestion
description: "This is a template priority class for all pods"
```

---

I am not a native english speaker myself, but it look good to me